parity/main.rs

```diff
@@ -81,11 +81,12 @@ Options:
 
   -j --jsonrpc             Enable the JSON-RPC API sever.
   --jsonrpc-url URL        Specify URL for JSON-RPC API server [default: 127.0.0.1:8545].
+  --jsonrpc-cors URL       Specify CORS header for JSON-RPC API responses [default: null].
```

Comment:
"null" means that `Access-Control-Allow-Origin` value will be `null`
https://www.w3.org/TR/cors/#access-control-allow-origin-response-header